src/components/settings-modal/settings-modal.scss

```diff
@@ -0,0 +1,88 @@
+@import '../../styles/variables';
+@import '../../styles/extends';
+@import '../../styles/mixins';
+
+.kui-theme--light {
+  --color-modal-bg: #{$color-bg-light-4};
+  --color-modal-text: #{$color-text-light};
+  --color-modal-icon: #{$color-text-light};
+  --color-modal-header-text: #{rgba($color-dark, 0.55)};
+  --color-toggle--on: #{$color-primary-2};
+  --color-toggle-bar--on: #{rgba($color-primary, 0.3)};
+  --color-toggle--off: #{$color-bg-dark-9};
+  --color-toggle-bar--off: #{rgba($color-bg-dark-9, 0.3)};
+}
+
+.kui-theme--dark {
+  --color-modal-bg: #{$color-bg-dark-8};
```

Comment:
This is more likely a question for @GabrielComymQB - I noticed that the settings modal is using a different background color and font size from the export modal - I understand that this is the color adopted from the design, however I noticed that visually the 2 modals do deviate quite a bit which perhaps isn't the best alignment visually given both are modals ( the large pipeline warning also uses a similar design language to the export modal). 

I wonder if we would need to either align with the background color of the old modal, or update the background color of the other modals for consistency? 

![image](https://user-images.githubusercontent.com/27775658/122533136-1fba6080-d019-11eb-87ea-c7c4864cb6a4.png)

![image](https://user-images.githubusercontent.com/27775658/122533392-6d36cd80-d019-11eb-8ab3-9f591ff035ad.png)

Reply:
I see that Kedro-UI has been updated and the relevant classes deleted, hence resolving this.